persistence/src/commonMain/db_user/com/wire/kalium/persistence/Clients.sq

```diff
@@ -21,10 +28,13 @@ deleteClientsOfUser:
 DELETE FROM Client WHERE user_id = ?;
 
 insertClient:
-INSERT INTO Client(user_id, id,device_type, is_valid)
-VALUES(?, ?,?, ?)
+INSERT INTO Client(user_id, id,device_type, client_type, is_valid, registration_date, label, model)
+VALUES(?, ?,?, ?, ?,?, ?, ?)
 ON CONFLICT(id, user_id) DO UPDATE SET
 device_type = coalesce(excluded.device_type, device_type),
+registration_date = coalesce(excluded.registration_date, registration_date),
+label = coalesce(excluded.label, label),
+model = coalesce(excluded.model, model),
```

Comment:
maybe add `is_verified = is_verified,` ???
to not lose the verification status.

Or there is an idea that if the device was updated then it's not the same device, so we can't be sure it's verified anymore 🤔

Reply:
Adding `Is_verified = is_verified` is redundant since this is the default behavior,  i did add a unit test to cover this case